packages/testing-kit/src/environment/configuration.ts

```diff
@@ -1,117 +1,61 @@
-import { unsafeEither } from "@marlowe.io/adapter/fp-ts";
-import * as E from "fp-ts/lib/Either.js";
-import { Blockfrost } from "lucid-cardano";
-import { SeedPhrase, seedPhrase } from "../wallet/seedPhrase.js";
-import {
-  RestClient,
-  mkRestClient,
-  RuntimeVersion,
-} from "@marlowe.io/runtime-rest-client";
-import * as Lucid from "lucid-cardano";
-import * as G from "@marlowe.io/runtime-rest-client/guards";
-import { MarloweJSON } from "@marlowe.io/adapter/codec";
-
-/**
- * Test Configuration : Read from an env file, it contains :
- * 1. The necessary configuration to run e2e tests over a Lucid Wallet and a Runtime.
- * 2. A Bank Wallet Seed Phrase to provision ephemeral Test Wallets
- */
-export type TestConfiguration = {
-  bank: { seedPhrase: SeedPhrase };
-  lucid: { blockfrost: Blockfrost; node: { network: Lucid.Network } };
-  runtime: {
-    version: RuntimeVersion;
-    url: URL;
-    client: RestClient;
-    node: { network: Network };
-  };
-};
-
-/**
- * Read Test Configurations from an env file
- * @returns
- */
-export const readEnvConfigurationFile =
-  async (): Promise<TestConfiguration> => {
-    const runtimeURL = readEnvRuntimeURL();
-    const lucidNetwork = readEnvLucidNodeNetwork();
-    const runtimeClient = mkRestClient(runtimeURL.toString());
-    const status = await runtimeClient.healthcheck();
-    const runtimeNodeNetwork = getNetwork(status.networkId);
-
-    if (!G.CompatibleRuntimeVersion.is(status.version)) {
-      throw {
-        message: "Runtime Version is not Compatible with the ts-sdk",
-        details: MarloweJSON.stringify(status),
-      };
-    }
-
-    const configuration = {
-      bank: { seedPhrase: readEnvBankSeedPhrase() },
-      lucid: {
-        blockfrost: readEnvBlockfrost(),
-        node: { network: toLucidNetwork(lucidNetwork) },
-      },
-      runtime: {
-        version: status.version,
-        url: runtimeURL,
-        client: mkRestClient(runtimeURL.toString()),
-        node: { network: runtimeNodeNetwork },
-      },
-    };
-    return configuration;
-  };
-
-const readEnvBlockfrost = (): Blockfrost => {
-  const { BLOCKFROST_URL, BLOCKFROST_PROJECT_ID } = process.env;
-  if (BLOCKFROST_URL == undefined)
-    throw "Test environment variable not defined (BLOCKFROST_URL)";
-  if (BLOCKFROST_PROJECT_ID == undefined)
-    throw "Test environment variable not defined (BLOCKFROST_PROJECT_ID)";
-  return new Blockfrost(BLOCKFROST_URL, BLOCKFROST_PROJECT_ID);
-};
+import * as t from "io-ts/lib/index.js";
+import { readFile } from "fs/promises";
+import * as fs from "fs";
+import * as path from "path";
+const lucidNetworkGuard = t.union([
+  t.literal("Mainnet"),
+  t.literal("Preview"),
+  t.literal("Preprod"),
+  t.literal("Custom"),
+]);
+
+export const testConfigurationGuard = t.type({
+  bank: t.type({ seedPhrase: t.string }),
+  lucid: t.type({
+    blockfrostProjectId: t.string,
+    blockfrostUrl: t.string,
+  }),
+  network: lucidNetworkGuard,
```

Comment:
We don't want necessary to have a lucid terminology exposed in our config file for the network... and this wasn't done yet , but we should check if the networks are compatible as well

Reply:
it is a union type of the words "Preprod", "Preview" and "Mainnet". I don't think it applies to "lucid terminology". And in any case it is used to setup a lucid wallet.